docs/api/covidcast-signals/nchs-mortality.md

```diff
@@ -0,0 +1,91 @@
+---
+title: NCHS Mortality Data
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# NCHS Mortality Data
+{: .no_toc}
+
+* **Source name:** `nchs-mortality`
+* **First issued:** Epiweek 50 2020 (6-12 December 2020)
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** state (see [geography coding docs](../covidcast_geography.md))
+* **License:** [NCHS Data Use Agreement](https://www.cdc.gov/nchs/data_access/restrictions.htm)
+
+This data source of national provisional death counts is based on death
+certificate data received and coded by the National Center for Health Statistics
+[(NCHS)](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm).
+
+| Signal | Description |
+| --- | --- |
+| `deaths_covid_incidence_num` | Number of weekly new deaths with confirmed or presumed COVID-19 |
+| `deaths_covid_incidence_prop` | Number of weekly new deaths with confirmed or presumed COVID-19, per 100,000 population |
+| `deaths_allcause_incidence_num` | Number of weekly new deaths from all causes |
+| `deaths_allcause_incidence_prop` | Number of weekly new deaths from all causes, per 100,000 population |
+| `deaths_flu_incidence_num` | Number of weekly new deaths involving Influenza |
+| `deaths_flu_incidence_prop` | Number of weekly new deaths involving Influenza, per 100,000 population |
```

Comment:
Number of weekly new deaths involving Influenza and pneumonia or deaths involving influenza and COVID-19, per 100, 000 population

Reply:
```suggestion
| `deaths_flu_incidence_prop` | Number of weekly new deaths involving Influenza and at least one of (Pneumonia, COVID-19), per 100,000 population |
```